crates/goose/src/providers/ollama.rs

```diff
@@ -51,9 +53,28 @@ impl OllamaProvider {
     }
 
     async fn post(&self, payload: Value) -> Result<Value, ProviderError> {
-        let url = format!("{}/v1/chat/completions", self.host.trim_end_matches('/'));
+        // OLLAMA_HOST is sometimes just the 'host' or 'host:port' without a scheme
+        let base = if self.host.starts_with("http://") || self.host.starts_with("https://") {
+            self.host.clone()
+        } else {
+            format!("http://{}", self.host)
```

Comment:
if i pass in `htp://` on accident this becomes `http://htp://`, pretty unintuitive behavior imo

i believe it'll error out below but not sure i like auto appending the scheme prefix as opposed to just erroring out to the user that they **must** set a http(s) scheme